docs/dev-guide/integration/rest-api/consumer/applicability.rst

```diff
@@ -95,11 +95,11 @@ URL and perform a GET request to retrieve the :ref:`task_group_summary`.
 
 :sample_request:`_` ::
 
- { 
+ {
   "repo_criteria": {
    "filters": {"id": {"$in": ["test-repo", "test-errata"]}},
-   "parallel": true
   }
+  "parallel": true
```

Comment:
I believe you need a comma between closing bracket and before `parallel: true`.
Comma after `filters` dictionary is not needed.